### DISCUSSION TITLE
How to specifically assign a specific USB Coral to Frigate (System has a total of two)

### DISCUSSION THREAD

**Discussion by dimatx** [blakeblackshear/frigate#9686] [Ask A Question]
My system has a total of 2 USB Corals, but I want to assign only one of the two to Frigate. I'm currently using docker-compose to deploy, and pass both USBs to Frigate via `- /dev/bus/usb:/dev/bus/usb`. However, this results in Frigate grabbing one of the two, but I'm not sure how to specific which of the two it should use.

**Comment by NickM-27** [comment]
you could adjust docker to only pass in one of the USB bus

**Reply by dimatx** [reply]
Do you know how? I was thinking the right way to do that may be "/dev/bus/usb/002/006:/dev/bus/usb/002/006", but the 006 changes on restarts, etc, so it's not reliable.
